resources/js/src/app/components/itemList/filter/ItemDropdown.js

```diff
@@ -0,0 +1,72 @@
+import url from '../../../mixins/url'
+
+Vue.component("item-dropdown", {
+    mixins: [url],
+
+    props: [
+        "template",
+        "facet"
+    ],
+
+    data: function () {
+        return {
+            isShowDropdown: false
+        }
+    },
+
+    created() {
+        this.$options.template = this.template || "#vue-item-dropdown";
+
+        window.addEventListener('click', this.onClick);
+    },
+
+    beforeDestroy() {
+        window.addEventListener('click', this.onClick);
+    },
+
+    computed: {
+        ...Vuex.mapState({
+            isLoading: state => state.itemList.isLoading
+        })
+    },
+
+    methods: {
+        getSelectedValue() {
+            let self = this;
+
+            let facetValue = this.facet.values.filter((value) => self.isValueSelected(self.facet.id, value.name));
+
+            if (facetValue.length === 0) {
+                return ''
+            }
+
+            return facetValue[0].name;
+        },
+
+        onClick(event) {
+            if (!this.$el.children[0].contains(event.target)) {
+                this.hideDropdown();
+            } else {
+                this.showDropdown();
+            }
+        },
+
+        selected: function (value) {
```

Comment:
This doesn't work. It doesn't remove the current filter and set the required filter correctly.